spring-cloud-gcp-samples/spring-cloud-gcp-integration-storage-sample/src/test/java/GcsSpringIntegrationTests.java

```diff
@@ -0,0 +1,102 @@
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+import java.util.stream.Collectors;
+
+import com.example.GcsSpringIntegrationApplication;
+import com.google.api.gax.paging.Page;
+import com.google.cloud.storage.Blob;
+import com.google.cloud.storage.BlobId;
+import com.google.cloud.storage.BlobInfo;
+import com.google.cloud.storage.Storage;
+import com.google.cloud.storage.StorageOptions;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.TestPropertySource;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.awaitility.Awaitility.await;
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assume.assumeThat;
+
+/**
+ * This test uploads a file to Google Cloud Storage and verifies that it was received in
+ * the local directory specified in application-test.properties.
+ */
+@RunWith(SpringRunner.class)
+@TestPropertySource("classpath:application-test.properties")
```

Comment:
We shouldn't need test properties. Just overwrite them via system properties in Travis config with `-Dgcs-read-bucket=...`.

Reply:
Done. I modified the test to use the application.properties file that the sample uses and these will be overridden by command line args in travis config.